cmd/snap-repair/runner.go

```diff
@@ -67,24 +82,145 @@ func (r *Repair) SetStatus(status RepairStatus) {
 
 // Run executes the repair script leaving execution trail files on disk.
 func (r *Repair) Run() error {
-	// XXX initial skeleton...
 	// write the script to disk
-	rundir := filepath.Join(dirs.SnapRepairRunDir, r.BrandID(), r.RepairID())
+	rundir := r.RunDir()
 	err := os.MkdirAll(rundir, 0775)
 	if err != nil {
 		return err
 	}
+
 	script := filepath.Join(rundir, fmt.Sprintf("script.r%d", r.Revision()))
-	err = osutil.AtomicWriteFile(script, r.Body(), 0600, 0)
+	err = osutil.AtomicWriteFile(script, r.Body(), 0700, 0)
+	if err != nil {
+		return err
+	}
+
+	// the date may be broken so we use an additional counter
+	baseName := fmt.Sprintf("r%d", r.Revision())
```

Comment:
we want to change the script and the assertion also to be:  r#.script  and r#.repair as well